tests/vmi_multus_test.go

```diff
@@ -857,8 +880,10 @@ var _ = Describe("SRIOV", func() {
 			configInterface(vmi2, "eth1", cidrB, "#")
 
 			// now check ICMP goes both ways
-			pingVirtualMachine(vmi1, cidrToIP(cidrA), "#")
-			pingVirtualMachine(vmi2, cidrToIP(cidrB), "#")
+			err = pingVirtualMachine(vmi1, cidrToIP(cidrA), "#")
+			Expect(err).ToNot(HaveOccurred(), "Failed to ping VMI %s within the given timeout", vmi1.Name)
```

Comment:
I think the optional string description is meant for you to specify the intended behavior of the test, not to print an error message when it doesn't. Check the example supplied on the `Assertion` [interface](https://github.com/kubevirt/kubevirt/blob/d072d7ffcbba8d862e79db7749fd0effe2ca2f4e/vendor/github.com/onsi/gomega/gomega_dsl.go#L327).

Updating the msg to 'VMI %s should be reachable' would do it.